python/tk_multi_loader/dialog.py

```diff
@@ -1180,16 +1201,26 @@ def _add_rdo_list_view_mod(self):
             self.ui.label_2.hide()
             self.ui.thumb_scale.hide()
 
-    def _add_rdo_publish_search(self):
+    def _add_rdo_status_filter(self, hlayout):
+        '''
+        Add a combo box to sort the latest publishes by status
+        Kind of copying off the _add_rdo_status_filter
+        '''
+        filter_label = QtGui.QLabel("Filter by Status")
```

Comment:
And I think the status filter near the search box looks meh..
![loader_status_filter2](https://cloud.githubusercontent.com/assets/2866068/8968446/473b079e-360b-11e5-960a-6cc333d1d9d5.png)